khal/utils.py

```diff
@@ -50,17 +50,15 @@ def generate_random_uid():
 
 
 def find_last_reset(string):
-    for match in re.finditer(ansi_reset, string):
-        pass
+    *_, match = re.finditer(ansi_reset, string)
```

Comment:
this will definitively break

Reply:
It should make `match` the last result of the iterator, like the original. Unless I'm missing something. I did a quick test when I was reviewing, but I didn't check against `re` specifically.

Reply:
but I believe for that two work, we need at least two matches, or not?

Reply:
The syntax seems to work for iterators with length 1. I'll double check `re` for good measure.

Reply:
Looks good.

Reply:
You are right, it totally works with iterators of length 1, learned something new, Thanks!

Unfortunately, it doesn't seem to work if there is no match at all, [some tests are failing therefore](https://travis-ci.com/github/pimutils/khal/jobs/504997519#L293).

Reply:
So I guess the options are:
- Revert these two changes
- Move the `try` to the `finditer` line and change the exception accordingly
- Replace the `try`/`except` with an `if match`...`else`

Thoughts?